lib/msal-common/src/utils/Constants.ts

```diff
@@ -81,7 +81,8 @@ export enum PersistentCacheKeys {
     ADAL_ID_TOKEN = "adal.idtoken",
     ERROR = "error",
     ERROR_DESC = "error.description",
-    ACTIVE_ACCOUNT = "active-account"
+    ACTIVE_ACCOUNT = "active-account",
+    ACTIVE_ACCOUNT_NEW = "active-account-new" //new cache entry for active_account for a more robust version for browser
```

Comment:
Instead of "new", can we describe it? For example:

```suggestion
    ACTIVE_ACCOUNT_NEW = "active-account-home-local-ids" //new cache entry for active_account for a more robust version for browser
```